src/dto/delete-task.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { Type } from 'class-transformer';
+import { IsInt, IsNumber, IsPositive } from 'class-validator';
+export class DeleteTaskDto {
```

Comment:
```suggestion
import { IsInt, IsNumber, IsPositive } from 'class-validator';

export class DeleteTaskDto {
```
Почему-то пробела нет между классом и импортом, правила ESLint точно работают ?)